contentcuration/contentcuration/models.py

```diff
@@ -958,6 +963,19 @@ def get_channel(self):
         except (ObjectDoesNotExist, MultipleObjectsReturned, AttributeError):
             return None
 
+    def get_thumbnail(self):
+        # Problems with json.loads, so use ast.literal_eval to get dict
+        if self.thumbnail_encoding:
+            thumbnail_data = ast.literal_eval(self.thumbnail_encoding)
```

Comment:
I did implement a function to handle this without relying on `ast.literal_eval`. Do you mind switching to [this](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/utils/parser.py#L108)?

Reply:
Sure, will do! :)

Reply:
Note that for channel thumbnails I migrated the thumbnail_encoding field to a JSON field to avoid this malarkey: https://github.com/learningequality/studio/pull/1140

The data migrations that I made to facilitate this should transparently transfer to the content node models also.

Reply:
Yes, I was thinking about doing this, but then I thought that if we go the route of migrating to all files for ContentNode thumbnails, we can just create a migration to remove thumbnail_encoding entirely on ContentNode. :)